package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "treemapper",
-  "version": "1.0.8",
+  "version": "1.0.9",
```

Comment:
Increasing release versions should not be part of feature branches, but should rather be done in release branches.